lib/datadog/core/configuration/option.rb

```diff
@@ -94,6 +96,93 @@ def default_precedence?
 
         private
 
+        def coerce_env_variable(value)
+          return context_exec(value, &@definition.env_parser) if @definition.env_parser
+
+          case @definition.type
+          when :int
+            # DEV-2.0: Change to a more strict coercion method. Integer(value).
+            value.to_i
+          when :float
+            # DEV-2.0: Change to a more strict coercion method. Float(value).
+            value.to_f
+          when :array
+            values = if value.include?(',')
+                       value.split(',')
+                     else
+                       value.split(' ') # rubocop:disable Style/RedundantArgument
+                     end
```

Comment:
> Since the default is not to support the spaces-variant?

Where is that defined? 

The code for `env_to_list` did not define any default value for `def env_to_list(var, default = [], comma_separated_only:, deprecation_warning: true)`

Doing a search on [`comma_separated_only`](https://github.com/search?q=repo%3ADataDog%2Fdd-trace-rb%20comma_separated_only&type=code) shows three instances of `true`, and four instances of `false`.

I'm happy to remove the if condition, I just to make sure to understand if we have to

Reply:
I mean the default for the DSL should be to not support it, because in my view the "spaces separator" is the weird use-case, so it should require extra work on the part of the settings that want it.

(As I mention below, accepting spaces on top of commas has the potential of creating more inconsistency across datadog libraries)